src/data/row.rs

```diff
@@ -56,6 +56,11 @@ impl Row {
                     _ => columns.iter().position(|target| target.value == name),
                 };
 
+                #[cfg(feature = "auto-increment")]
+                if column_def.is_auto_incremented() {
+                    return Ok(Value::Null);
+                }
```

Comment:
Though auto increment is applied to column, it should not block adding custom value.

Reply:
Looks like you're right! I **do** need to take it more slowly.

Not sure how I managed to let this happen.

(This should only be happening if value unset)